lms/envs/common.py

```diff
@@ -2219,14 +2221,6 @@
 ENABLE_CREDIT_ELIGIBILITY = True
 FEATURES['ENABLE_CREDIT_ELIGIBILITY'] = ENABLE_CREDIT_ELIGIBILITY
 
-################ Enable JWT auth ####################
-# When this feature flag is set to False, API endpoints using
-# JSONWebTokenAuthentication will reject requests using JWT to authenticate,
-# even if those tokens are valid. Set this to True only if you need those
-# endpoints, and have configured settings 'JWT_AUTH' to override its default
-# values with secure values.
-FEATURES['ENABLE_JWT_AUTH'] = False
```

Comment:
the PR that added this, https://github.com/edx/edx-platform/pull/11599, contains a lot of the context about why it went in in the first place.  Have there been subsequent conversations to determine we don't need it anymore?